src/doc/rustc/src/codegen-options/index.md

```diff
@@ -530,6 +530,21 @@ This also supports the feature `+crt-static` and `-crt-static` to control
 Each target and [`target-cpu`](#target-cpu) has a default set of enabled
 features.
 
+## tune-cpu
+
+This instructs `rustc` to schedule code specifically for a particular processor.
+This does not affect what instruction sets are used or the compatibility, but
+might make your code run a bit faster on the selected CPU.
+
+You can run `rustc --print target-cpus` to see the valid options to pass
+here. Additionally, `native` can be passed to use the processor of the host
```

Comment:
Might also be a good idea to explain `generic` in here and `target-cpu`: it's basically a profile for ancient instructions tuned for modern CPU.